src/general/HelpScreen.tscn

```diff
@@ -93,10 +93,12 @@ SkipOverridingFocusForElements = [NodePath("../../../..")]
 [node name="TipMessageLabel" type="Label" parent="CenterContainer/VBoxContainer"]
 editor_description = "PLACEHOLDER"
 modulate = Color(1, 1, 0, 1)
+custom_minimum_size = Vector2(812, 20)
```

Comment:
Does the minimum size need to be so extreme? I worry that if somebody wants to embed the message in another place or the help menu, this is going to be problematic. I usually set just some kind of minimum sensible size like `100, 0` or `150, 0` for line wrapped labels, and make sure fill / expand is on so that it can take advantage of the available space.